README.md

```diff
@@ -32,35 +32,40 @@ Cron is an application which will execute commands on a defined schedule. WebFac
 
+### SFTP
+
+If you're not happy navigating around your server's folders and files through SSH, you might find some of this process easier if you access your server with an FTP client over Secure FTP. WebFaction [also has this covered](https://docs.webfaction.com/user-guide/access.html#connecting-with-ftp).
+
 ## Installation
 
 This utility works on [CentOS 6 and 7 boxes](https://docs.webfaction.com/user-guide/server.html#finding-your-server-s-operating-system). The CentOS 5 systems do not have a new enough OpenSSL to include the algorithms required. You may be able to make this work using rbenv and compiling openssl yourself. A tutorial for CentOS 5 is available here: https://github.com/will-in-wi/letsencrypt-webfaction/wiki/Install-custom-OpenSSL-and-Ruby-on-CentOS-5-host
 
-All places where you need to substitute a value specific to your setup will be denoted with square brackets, e.g. `[mydomain.tld]`. There are cases where shell variables are used, such as `$HOME`. These should be typed verbatim.
+All places where you need to substitute a value specific to your setup will be denoted with square brackets, e.g. [yourdomain.com]. There are cases where shell variables are used, such as `$HOME`. These should be typed verbatim.
 
-You can install LetsEncrypt Webfaction using the system Ruby or using RBEnv.
+You can install LetsEncrypt WebFaction using the system Ruby or using RBEnv.
 
 ### System Ruby
 
 This is the simpler method and is preferred.
 
-Run the following command to install the letsencrypt_webfaction package via the [RubyGems package management site](https://rubygems.org/gems/letsencrypt_webfaction):
+Run the following command in an SSH session to install the letsencrypt_webfaction package via the [RubyGems package management site](https://rubygems.org/gems/letsencrypt_webfaction):
 
 ```sh
 GEM_HOME=$HOME/.letsencrypt_webfaction/gems RUBYLIB=$GEM_HOME/lib gem2.2 install letsencrypt_webfaction
 ```
 
-Add the following to `~/.bash_profile` to simplify the running of LetsEncrypt Webfaction with a bash function:
+Add the following to `~/.bash_profile` (using, for example, an FTP client or your favorite text editor):
 
 ```sh
 function letsencrypt_webfaction {
     PATH=$PATH:$GEM_HOME/bin GEM_HOME=$HOME/.letsencrypt_webfaction/gems RUBYLIB=$GEM_HOME/lib ruby2.2 $HOME/.letsencrypt_webfaction/gems/bin/letsencrypt_webfaction $*
 }
 ```
+This will simplify the running of the LetsEncrypt WebFaction command, by setting some variables in advance.
 
-Then run the command `source $HOME/.bash_profile` to apply the new settings.
+After saving `~/.bash_profile`, run the command `source $HOME/.bash_profile` to apply the new settings.
 
-Now, you can run `letsencrypt_webfaction` from the shell to get certificates. You can run this from any folder.
+Now, you are ready to run `letsencrypt_webfaction` from your SSH session to get certificates. See below for usage. 
```

Comment:
Sheesh, you'd think I hadn't written decent prose in a while. This flows much better.